nilearn/plotting/js_plotting_utils.py

```diff
@@ -99,7 +110,12 @@ def __init__(self, html, width=600, height=400):
 
     def _check_n_open(self):
         HTMLDocument._all_open_html_repr.add(self)
-        if len(HTMLDocument._all_open_html_repr) > 9:
+        if MAX_IMG_VIEWS_BEFORE_WARNING is None:
+            return
+        if MAX_IMG_VIEWS_BEFORE_WARNING < 0:
+            return
+        if len(HTMLDocument._all_open_html_repr
```

Comment:
Can we do:
```python
         if len(HTMLDocument._all_open_html_repr) ... :
                  # display warning
         else:
                  return
```

Reply:
we need to check if `MAX_IMG_VIEWS_BEFORE_WARNING` is not `None` before comparing it to an int. also, this makes it more explicit in which cases the check is not performed